src/unity/python/turicreate/toolkits/drawing_recognition/drawing_recognition.py

```diff
@@ -0,0 +1,480 @@
+from turicreate.toolkits import evaluation as _evaluation
+import turicreate.toolkits._internal_utils as _tkutl
+from ._sframe_loader import SFrameRecognitionIter as _SFrameRecognitionIter
+from .. import _mxnet_utils
+from turicreate import extensions as _extensions
+from copy import copy as _copy
+
+BITMAP_WIDTH = 28
+BITMAP_HEIGHT = 28
+
+class Model(_HybridBlock):
+    def __init__(self, num_classes, **kwargs):
+        super(Model, self).__init__(**kwargs)
+        with self.name_scope():
+            # layers created in name_scope will inherit name space
+            # from parent layer.
+            self.conv1 = _nn.Conv2D(channels=16, kernel_size=(3,3), 
+                                    padding=(1,1), activation='relu')
+            self.pool1 = _nn.MaxPool2D(pool_size=(2,2))
+            self.conv2 = _nn.Conv2D(channels=32, kernel_size=(3,3), 
+                                    padding=(1,1), activation='relu')
+            self.pool2 = _nn.MaxPool2D(pool_size=(2,2))
+            self.conv3 = _nn.Conv2D(channels=64, kernel_size=(3,3), 
+                                    padding=(1,1), activation='relu')
+            self.pool3 = _nn.MaxPool2D(pool_size=(2,2))
+            self.flatten = _nn.Flatten()
+            self.fc1 = _nn.Dense(units=128, activation='relu')
+            self.fc2 = _nn.Dense(units=num_classes, activation=None)
+
+    def hybrid_forward(self, F, x):
+        x = self.pool1(self.conv1(x))
+        x = self.pool2(self.conv2(x))
+        x = self.pool3(self.conv3(x))
+        x = self.flatten(x)
+        x = self.fc1(x)
+        x = self.fc2(x)
+        return F.softmax(x)
+
+def create(input_dataset, annotations=None, num_epochs=100, feature="bitmap", model=None,
+           classes=None, batch_size=256, max_iterations=0, verbose=True, 
+           **kwargs):
+    """
+    Create a :class:`DrawingRecognition` model.
+    """
+    start_time = _time.time()
+
+    is_stroke_input = (input_dataset[feature].dtype != _tc.Image)
+
+    if is_stroke_input:
+        # This only works on macOS right now
+        dataset = _extensions._drawing_recognition_prepare_data(
+            input_dataset, "bitmap", "label")
+    else:
+        dataset = input_dataset
+
+    column_names = ['Iteration', 'Loss', 'Elapsed Time']
+    num_columns = len(column_names)
+    column_width = max(map(lambda x: len(x), column_names)) + 2
+    hr = '+' + '+'.join(['-' * column_width] * num_columns) + '+'
+
+    progress = {'smoothed_loss': None, 'last_time': 0}
+    iteration = 0
+
+    if classes is None:
+        classes = dataset['label'].unique()
+    classes = sorted(classes)
+    class_to_index = {name: index for index, name in enumerate(classes)}
+
+    def update_progress(cur_loss, iteration):
+        iteration_base1 = iteration + 1
+        if progress['smoothed_loss'] is None:
+            progress['smoothed_loss'] = cur_loss
+        else:
+            progress['smoothed_loss'] = (0.9 * progress['smoothed_loss'] 
```

Comment:
If I'm not mistaken, smoothed loss in OD was again a concession to the length of an epoch. Most of our other toolkits yield statistics at the granularity of training step. I.e., what was the average loss during epoch n? There's also no reason we shouldn't output metrics like accuracy. (This could be a followup PR though)

Reply:
Noted. Will make a followup PR to address this. Tracking this issue in #1464